apps/examples/board_specific_demo/ameba_secure_storage_demo/README.txt

```diff
@@ -7,6 +7,10 @@ examples/secure_storage
   This shows how to call write and read from Secure Storage with SE APIs,
   and how to read data written to flash.
 
+  This Example writes a user given byte of data (or default) to all bytes
+  of a user given slot (or default) of the Secure Storage
+
+  Usage: - sstorage -s <slot index> -d < hex data value>
```

Comment:
Can you pls remove `-` sign before `sstorage`?

Reply:
Understood, hyphen created confusion